ApollosProject/apollos-apps
# fix: add header avoidance for android on the user settings page

This PR adds a calculated padding to the top of the Nav stack on the Following Me and Followed By Me pages on the new User Settings page. The logic here is pulled almost directly from BackgroundView and [this PR](https://github.com/ApollosProject/apollos-apps/pull/2510), with the main difference being that we cannot apply this header to a view directly and need to do it via `screenOptions` in the Navigator.

I tested this on a Galaxy S22 and a Pixel 6 sim.

|Before|After|
|---|---|
|![Screenshot_1659709754](https://user-images.githubusercontent.com/72768221/183124764-22e6d49c-431f-4116-968f-d65828ee6283.png)|![Screenshot_1659718105](https://user-images.githubusercontent.com/72768221/183124781-bda809b1-7a5d-419a-aa01-de5f3765fb9a.png)|

## Testing
1. Pull down this repo
2. Run the setup script `./scripts/setup-app.sh`. Define CHURCH and APOLLOS_API_KEY
3. Go to `deploys/mobile` and run `yarn start` and `yarn android`
4. Navigate to the User Settings page and check out the changes on the Followed By Me page
